src/styles/blocks/calendar.scss

```
@@ -0,0 +1,55 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  padding: $calendar-padding;
+  gap: 1px;
+  width: calendarWidth($day-size, $calendar-padding);
+
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    @include square($day-size);
+
+    border: 1px solid black;
+    background-color: #eee;
+    transition: 0.5s;
```

It's better to use accurate values
```suggestion
    transition: 0.5s prop1 ease, 0.5s prop2 ease;
```